release/readme.md

```diff
@@ -67,7 +67,8 @@
 ## Publish the release
 
 * add hashes in `install.sh` (and check signatures)
-* publish opam packages in opam-repository (and add `flags: avoid-version` and `available: opam-version >= "2.1.0"` to each packages if this is not a stable version)
+* `opam publish --exclude=opam,opam-admin https://github.com/ocaml/opam/archive/refs/tags/<tag>.tar.gz`
+  (and add `flags: avoid-version` and `available: opam-version >= "2.1.0"` to each packages if this is not a stable version)
```

Comment:
Is that worth having a feature in opam-publish for? (`--pre-release` or some such?)

Reply:
Yes, definitely. I did think about it but forgot to open a ticket. Done in https://github.com/ocaml-opam/opam-publish/issues/163